examples/http-service-and-resource-paths/http_service_and_resource_paths.md

```diff
@@ -1,6 +1,6 @@
 # REST service - Service and resource paths
 
-Ballerina supports writing RESTful services. Each resource method can be invoked via `service-path` and `resource-path`. The `resource accessor` confines the resource to the specified HTTP method.
+The `service-path` is defined in the service declaration and the `resource-path` is defined in the resource method definition. Each resource can be invoked by using the `service-path`, `resource-path`, and `resource-accessor`. In an HTTP resource, `resource-accessor` confines the resource to a specific HTTP method.
```

Comment:
@praneesha Do we need these dashes?
Eg: `service-path`, `resource-path`, and `resource-accessor`

Reply:
what does grammerly say?

Reply:
Fixed with [c265a1c99d8e45200bf8916d542242e3012c417a](https://github.com/ballerina-platform/ballerina-distribution/pull/3764/commits/c265a1c99d8e45200bf8916d542242e3012c417a)

Reply:
@shafreenAnfar @ayeshLK @chamil321 We don't need the dash if it is used in the noun form (e..g. The service path is defined in the service declaration.).

We need the dash when it is referred to as a variable (e.g. The `service-path` property is defined in the service declaration.).